wakes.gemspec

```diff
@@ -26,7 +26,8 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.test_files = Dir['spec/**/*']
 
-  spec.add_dependency 'google-api-client'
+  spec.add_dependency 'google-apis-analyticsdata_v1beta', '~> 0.1'
```

Comment:
@benhutton Interesting. So they are still on a beta version!

Reply:
yeah, this is all pretty new!